INSTALL.md

```diff
@@ -1312,8 +1314,8 @@ Package builders who want to configure the library for standard locations,
 but have the package installed somewhere else so that it can easily be
 packaged, can use
 
-  $ make DESTDIR=/tmp/package-root install         # Unix
-  $ mms/macro="DESTDIR=TMP:[PACKAGE-ROOT]" install ! OpenVMS
+    make DESTDIR=/tmp/package-root install         # Unix
+    mms/macro="DESTDIR=TMP:[PACKAGE-ROOT]" install ! OpenVMS
```

Comment:
Spurious extra space at the beginning (5 spaces, rather than 4)

Reply:
This is just a diff artifact - indentation is 4 as intended.